release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -629,6 +618,11 @@ In the following tables, features are marked with the following statuses:
 |General Availability
 |General Availability
 
+|{ibmpowerProductName} Virtual Server Block CSI Driver Operator
+|Not Available
+|Technology Preview
+|Technology Preview
```

Comment:
Might be GA for 4.14?

Reply:
I saw that message, but I'll wait till we know to change it.